packages/account/src/providers/utils/handle-gql-error-message.ts

```diff
@@ -0,0 +1,20 @@
+import { ErrorCode, FuelError } from '@fuel-ts/errors';
+import type { GraphQLError } from 'graphql';
+
+export enum GqlErrorMessage {
+  NOT_ENOUGH_COINS = 'not enough coins to fit the target',
+}
+
+export const handleGqlErrorMessage = (errorMessage: string, rawError: GraphQLError) => {
+  switch (errorMessage) {
+    case GqlErrorMessage.NOT_ENOUGH_COINS:
+      throw new FuelError(
+        ErrorCode.NOT_ENOUGH_FUNDS,
+        'The transaction does not have enough funds to cover its execution.',
```

Comment:
@Dhaiwat10 what do you think about: 

`The account does not have enough funds to cover the transaction execution.`?

Since `not enough coins to fit the target`error happens when we call `getResourcesToSpend` informing an amount higher than the one owned by the account.

Reply:
@Torres-ssf that was the message I used initially but the team asked for a change here: https://github.com/FuelLabs/fuels-ts/pull/2902#discussion_r1714965751